docs/modules/gear/consumables.mdx

```diff
@@ -0,0 +1,68 @@
+---
+id: consumables
+title: Consumables
+---
+
+Consumables are items that can be made "edible" to the player and "consuming" it will allow them to run an action or receive a kit.
+It can be defined and applied to items in [kits](/docs/modules/gear/kits).
+When consumed, these items can trigger [actions](/docs/modules/mechanics/actions-triggers), and override vanilla eating or drinking behavior.
+
+#### Consumables Element
+
+<div className="table-container">
+  | Element | Description |
+  |---|---|
+  | `<consumables> </consumables>` | Node containing the consumables definitions. |
+
+  | Sub-elements ||
+  |---|---|
+  | `<consumable> </consumable>` | An individual consumable. |
+</div>
+
+#### Consumable Attributes
+
+<div className="table-container">
+  | Attribute | Description | Value | Default |
+  |---|---|---|---|
+  | `id` | <span className="badge badge--danger">Required</span>Unique identifier used to reference this consumable from other places in the XML. | <span className="badge badge--primary">String</span> |
+  | `action` | <span className="badge badge--danger">Required</span>Run the specified action upon consumption. | [Action ID](/docs/modules/mechanics/actions-triggers) |
+  | `kit` | The kit to give upon consumption. | [Kit ID](/docs/modules/gear/kits) |
```

Comment:
Is mentioning the kit the same concern brought up by @Pablete1234 in #90? I have no issue with this being in the docs but afaik it's an alias to an Action switch we're trying to switch over from.

Reply:
should've probably replied to this earlier, but yeah, i agree
for newer modules we shouldn't encourage 2 standards, kit and action are mutually exclusive, you can only have one of the two, and action is the more generic one that can take in a kit too, i'd kill kit here

Reply:
Just had a chance to review this on a map using consumable, seems like that is the case. Instead of creating a new badge like what the original pull request was doing, could we set a precedent from https://pgm.dev/docs/modules/objectives/ctf#net-attributes and https://pgm.dev/docs/modules/gear/crafting#shaped-recipes and display it as `action | kit` to show that they're aliases?

Reply:
I guess we could use that syntax yeah